### ISSUE TITLE
Review "hero-image" accessibility.

### ISSUE THREAD

**Issue by notlee** [Financial-Times/origami#219]
That "hero image" teaser variant was added about 4 years ago. Due to the
`has-image` modifier the naming is a little confusing. This variant behaves
like a hero teaser on smaller viewports but on larger viewports the image
is much larger and the copy of the teaser overlays the image, like a poster.

Is it accessible? I suspect depending on the image it is not due to low 
contrast and should be updated.

![Screenshot 2020-11-16 at 11 01 36](https://user-images.githubusercontent.com/10405691/99245525-9f6e1b80-27fb-11eb-817e-9b306724517b.png)

**Comment by notlee** [comment]
We're fixing some issues for this which came up in a recent audit
